docs/source/api/v3/topologies_name_queue_update.rst

```diff
@@ -0,0 +1,86 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-topologies-name-queue_update:
+
+****************************
+``topologies/{{name}}/queue_update``
+****************************
+
+``POST``
+========
+:term:`Queue` or "dequeue" updates for all servers assigned to the :term:`Cachegroups` in a specific :term:`Topology`.
```

Comment:
Get a build warning:
`WARNING: term not in glossary: Cachegroups`

Reply:
Changed term reference to `Cache Groups` in 3b2956594